integration_tests/benchmarks/layers/layers_benchmarks.ts

```diff
@@ -27,6 +27,7 @@ import * as tfl from '@tensorflow/tfjs-layers';
 import {BrowserEnvironmentType, BrowserEnvironmentInfo, ModelBenchmarkRun, ModelFunctionName, ModelTrainingBenchmarkRun, VersionSet, PythonEnvironmentInfo, NodeEnvironmentType, NodeEnvironmentInfo, EnvironmentInfo} from '../types';
 // tslint:disable-next-line:max-line-length
 import {addEnvironmentInfoToFirestore, addOrGetTaskId, addBenchmarkRunsToFirestore, addVersionSetToFirestore} from '../firestore';
+import {NamedTensorMap} from '@tensorflow/tfjs-converter/dist/src/data/types';
```

Comment:
maybe rename the file to model_benchmarks.ts and move to a different directory?

Reply:
Good point. Done. Renamed to `model_benchmarks.ts` and moved one level up.